src/panels/lovelace/cards/hui-entities-card.js

```diff
@@ -114,22 +113,12 @@ class HuiEntitiesCard extends EventsMixin(PolymerElement) {
 
     this._elements = [];
 
-    for (let i = 0; i < entities.length; i++) {
-      const entity = entities[i];
+    // eslint-disable-next-line
```

Comment:
What is being disabled here?

Reply:
The guide does not like the for of

Reply:
`entities.forEach((entity) => { ... });`

Reply:
I am aware of that but imo for of is ok. I'll leave the call to @balloob

Reply:
Why does the linter not like it? What is the reasoning. We are not going to disable the application of each rule at each usage of for of. See what the linter says and then we can use that to decide if we should disable the rule or not use for of

Reply:
https://github.com/airbnb/javascript/issues/1271